src/main/java/de/tk/opensource/privacyproxy/config/CookieNameMatchType.java

```diff
@@ -1,10 +1,6 @@
-/*--- (C) 1999-2019 Techniker Krankenkasse ---*/
-
 package de.tk.opensource.privacyproxy.config;
 
 public enum CookieNameMatchType {
 
-	PREFIX, FULL
-}
-
-/*--- Formatiert nach TK Code Konventionen vom 05.03.2002 ---*/
+    PREFIX, FULL
+}
```

Comment:
In einigen Dateien fehlt jetzt die trailing newline, ist bei Java-Klassen wohl nicht so schlimm, aber für einige UNIX-Tools kann das Probleme machen, daher würd ich die immer mitnehmen

Reply:
Gut gesehen, hab diese wieder hinzugefügt